### PR TITLE
fixed safe delete problems

### PR DESCRIPTION
**Pull request by Feazom**

### CODE REVIEW

**Comment by IlyaBat9** [comment]
https://lamart.atlassian.net/browse/SCHOOL-464

**Comment by IlyaBat9** [comment]
SCHOOL-464

File: open_schools_platform/common/admin.py [hunk 1]
```diff
@@ -29,17 +34,30 @@ def choices(self, changelist):
...
+def admin_wrapper(model: Type[BaseModel]):
+    def wrap_admin_model(admin_model: Type[BaseAdmin]):
+        delete_policy = model._safedelete_policy
+        if delete_policy:
+            if delete_policy == SOFT_DELETE or delete_policy == SOFT_DELETE_CASCADE:
+                admin_model.actions = ('undelete_selected',)
+                admin_model.highlight_deleted_field.short_description = admin_model.field_to_highlight  # type: ignore[attr-defined] # noqa
+
+                if admin_model.list_display:
+                    admin_model.list_display = BaseAdmin.list_display + admin_model.list_display  # type: ignore[operator] # noqa
```
Comment (Ivan228777) [inline open_schools_platform/common/admin.py:50]: Can you please explain why we need "+" operator here? Are we merging list_displays this way?

Comment (Feazom) [reply open_schools_platform/common/admin.py:50]: This is for concatenating the values of the field that were written in the base class

**Review by Ivan228777 — APPROVED** [review]

**Review by zavarin-michael — APPROVED** [review]

**Pull request merged**